packages/@react-facet/dom-fiber/src/types.ts

```diff
@@ -37,7 +37,8 @@ export interface NoTimeout {
 
 export type FocusCallback = (e: FocusEvent) => void
 export type TouchCallback = (e: TouchEvent) => void
-export type MouseCallback = (event: MouseEvent) => void
+export type MouseCallback = (e: MouseEvent) => void
+export type ScrollCallback = (e: Event) => void
```

Comment:
Should be `UIEvent` instead of a regular `Event`. See this for more details: https://developer.mozilla.org/en-US/docs/Web/API/GlobalEventHandlers/onscroll#value

Reply:
Yep! Thanks for spotting it. Here is the fix [d1ad801](https://github.com/Mojang/ore-ui/pull/61/commits/d1ad8013b10f10a44b95062c67906a7fc6f4c455)

Reply:
Hmm, I got a type error. Could it be that the type definition on TypeScript is wrong? https://github.com/microsoft/TypeScript/blob/main/lib/lib.dom.d.ts#L5618 

```
"scroll": Event;
```

Reply:
That is really weird! The type definition indeed seems to be wrong on TypeScript's side. We can either cast the handlers or just go with `Event`. I would vote for cast to the correct type. It's not a huge thing though, so go with whatever you think is best.